proto/greenfield/storage/common.proto

```diff
@@ -110,3 +110,12 @@ message LocalVirtualGroup {
   // Notice that the minimum unit of charge is 128K
   uint64 total_charge_size = 4;
 }
+
+message BucketFlowRateLimit {
```

Comment:
`BucketFlowRateLimit` is just `Int`. Why we need this instead of using `Int`?

Reply:
we can add another fields